addons/forum-time-zones/addon.json

```diff
@@ -1,16 +1,58 @@
 {
-  "name": "Local timezone on forums",
-  "description": "Uses your local time zone for displaying post dates and times on the forums.",
+  "name": "Forum time changer",
+  "description": "Changes the times for posts in the forums.",
   "userscripts": [
     {
-      "url": "userscript.js",
+      "url": "timezones.js",
       "matches": ["topics"],
       "runAtComplete": false
+    },
+    {
+      "url": "matcher.js",
+      "matches": ["topics"],
+      "runAtComplete": false
+    }
+  ],
+  "credits": [
+    {
+      "name": "god286/webdev03",
+      "link": "https://github.com/webdev03"
+    },
+    {
+      "name": "_nix",
+      "link": "https://github.com/towerofnix"
     }
   ],
   "dynamicEnable": true,
   "dynamicDisable": true,
   "versionAdded": "1.0.0",
   "tags": ["forums", "community"],
+  "settings": [
+    {
+      "name": "Use local timezone",
+      "id": "local-timezone",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Use twelve hour time",
+      "id": "twelve-hour-time",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Use relative dates",
+      "id": "relative-dates",
+      "type": "boolean",
+      "default": true
+    }
```

Comment:
This only works if twelve-hour-time is enabled as well

Reply:
Do relative dates also use relative time? For example, is "8 hours ago" an expected relative date? What about "yesterday 08:00 AM"?

Reply:
No, relative dates are only for the date currently. With all options on, "Yesterday 00:50:07" becomes "Yesterday at 12:50:07 am"

Reply:
This setting "relative-dates" only works when 12 hour time is on. I need to fix that soon...

Reply:
Setting this conversation as resolved then.